tasks/main.yml

```diff
@@ -6,6 +6,10 @@
   include: pkg_url.yml
   when: threatstack_pkg_url is undefined
 
+- name: Disable auditd service
+  include: disable_auditd.yml
```

Comment:
I feel like since it's just one task it probably doesn't need its own file, we can just put the task here. Either way works though.